packages/syft/src/syft/core/node/new/api.py

```diff
@@ -73,12 +73,11 @@ class APIEndpoint(SyftBaseObject):
     pre_kwargs: Optional[Dict[str, Any]]
 
 
-@serializable(recursive_serde=True)
+@serializable(attrs=["signature", "credentials", "serialized_message"])
```

Comment:
Alternatively this can be `@serializable(without=["cached_deseralized_message"])` if the attrs list grows too big